drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SettingsPresenter.java

```diff
@@ -97,30 +104,29 @@ public void reset() {
     protected void setRuleSettings(SimulationDescriptor simulationDescriptor) {
         view.getDmnSettings().getStyle().setDisplay(Style.Display.NONE);
         view.getRuleSettings().getStyle().setDisplay(Style.Display.INLINE);
-        view.getDmoSession().setValue(simulationDescriptor.getDmoSession());
-        view.getKieBase().setValue(simulationDescriptor.getKieBase());
-        view.getKieSession().setValue(simulationDescriptor.getKieSession());
-        view.getRuleFlowGroup().setValue(simulationDescriptor.getRuleFlowGroup());
+        view.getKieSession().setValue(Optional.ofNullable(simulationDescriptor.getDmoSession()).orElse(""));
```

Comment:
@danielezonca 
Please keep the model name and the view' one in synchro: if the property to se/get in the model is "dmoSession", the element in the view should have the same name.

Reply:
I cannot do it completely because this parameter **is** `KieSession` and not `dmoSession`. I can update the view but the label to show the user has to be `KieSession` so we can only decide where to break the consistency.
I have updated the view as requested but the proper way to fix it is to update the model too and migrate the files but it is too much effort to do now.